src/SOS/SOS.UnitTests/Scripts/GCPOH.script

```diff
@@ -39,7 +39,7 @@ VERIFY:.*Thread <HEXVAL>:
 VERIFY:\s+<HEXVAL>\s+<HEXVAL>\s+GCPOH\.Main\(\)\s+\[.*[Gg][Cc][Pp][Oo][Hh]\.cs\s+@\s+22\]\s+
 
 SOSCOMMAND:VerifyHeap
-VERIFY:\s*No heap corruption detected.\s*
+VERIFY:.* 0 errors.*
```

Comment:
Isn't "No heap corruption detected" a better message for most users?

Reply:
Good idea.  I will add that back as an extra line if we find no errors.